### ISSUE TITLE
Unary minus doesn't work for non-literals

### ISSUE THREAD

**Issue by masonproffitt** [iris-hep/qastle#28]
Example:
```python
qastle.python_source_to_text_ast('-a')
SyntaxError: Unsupported unary - operand type: <class '_ast.Name'>
```
And it can't be implemented easily as a UnaryOp because there's no way to distinguish between a `USub` and `Sub` when a composite expression starts with `(-` unless you look at the number of fields first.

**Comment by masonproffitt** [comment]
Lisp handles this by indeed just looking at the number of elements (http://jtra.cz/stuff/lisp/sclr/_minus.html). If there's only one, it does negation, otherwise it uses the first element as the subtrahend. This is probably the way to go.

**Comment by masonproffitt** [comment]
fixed by #29
